opencensus/trace/ext/httplib/trace.py

```diff
@@ -57,7 +57,7 @@ def wrap_httplib_request(request_func):
     """Wrap the httplib request function to trace. Create a new span and update
     and close the span in the response later.
     """
-    def call(self, method, url, *args, **kwargs):
```

Comment:
I think adding this feature requires new unit tests

Reply:
updated unit test to reflect the change